go/enclave/events/subscription_manager.go

```diff
@@ -179,3 +182,57 @@ func isRelevant(userAddrs []string, account *gethcommon.Address) bool {
 
 	return false
 }
+
+// Applies `filterLogs`, below, to determine whether the log should be filtered out.
```

Comment:
Maybe make it clearer here that these are from the user. Like user-submitted filter or subscription filter or something.